test/functionalTest/cases/950_GET_v2_entity/GET_v2_entity_too_many.test

```diff
@@ -0,0 +1,159 @@
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+GET Entities V2 with Pagination
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# 01. POST /v1/updateContext, creating 5 entities
+# 02. GET /v2/entities/E1
+#
+
+echo "01. POST /v1/updateContext"
+echo "=========================="
+payload='{
+  "contextElements": [
+    {
+      "id":   "E1",
+      "type": "T1"
+    },
+    {
+      "id":   "E1",
+      "type": "T2"
+    },
+    {
+      "id":   "E1",
+      "type": "T3"
+    },
+    {
+      "id":   "E1",
+      "type": "T4"
+    },
+    {
+      "id":   "E1",
+      "type": "T5"
+    }
+  ],
+  "updateAction": "APPEND"
+}'
+orionCurl --url /v1/updateContext --payload "$payload" --json
+echo
+echo
+
+echo "02. GET /v2/entities/E1"
+echo "============================="
+orionCurl --url '/v2/entities/E1' --json
+echo
+echo
+
+--REGEXPECT--
+01. POST /v1/updateContext
+==========================
+HTTP/1.1 200 OK
+Content-Length: 1072
+Content-Type: application/json
+Date: REGEX(.*)
+
+{
+    "contextResponses": [
+        {
+            "contextElement": {
+                "id": "E1",
+                "isPattern": "false",
+                "type": "T1"
+            },
+            "statusCode": {
+                "code": "200",
+                "reasonPhrase": "OK"
+            }
+        },
+        {
+            "contextElement": {
+                "id": "E1",
+                "isPattern": "false",
+                "type": "T2"
+            },
+            "statusCode": {
+                "code": "200",
+                "reasonPhrase": "OK"
+            }
+        },
+        {
+            "contextElement": {
+                "id": "E1",
+                "isPattern": "false",
+                "type": "T3"
+            },
+            "statusCode": {
+                "code": "200",
+                "reasonPhrase": "OK"
+            }
+        },
+        {
+            "contextElement": {
+                "id": "E1",
+                "isPattern": "false",
+                "type": "T4"
+            },
+            "statusCode": {
+                "code": "200",
+                "reasonPhrase": "OK"
+            }
+        },
+        {
+            "contextElement": {
+                "id": "E1",
+                "isPattern": "false",
+                "type": "T5"
+            },
+            "statusCode": {
+                "code": "200",
+                "reasonPhrase": "OK"
+            }
+        }
+    ]
+}
+
+
+02. GET /v2/entities/E1
+=============================
+HTTP/1.1 409 Conflict
+Content-Length: 73
+Content-Type: application/json
+Date: REGEX(.*)
+
+{
+    "description": "/v2/entities?id=E1",
```

Comment:
I would suggest:

"There is more than one entity with E1 as Id, you can get the list of all the entities invoking: GET /v2/entities?id=E1"

I think that suggesting refining with type is not a "complete" hint: even adding type it could be a conflict (due to service paths), so probably it is better not trying to be too much clever from our side :)

Do you really think spec should include the exact text of descriptions? I think that error messages (e.g. "NotFound, etc.) should be included in the spec because client could make decission based on their values, but descriptions are text for humans and could be left free to API implementations.